### DISCUSSION TITLE
ct web

### DISCUSSION THREAD

**Discussion by jbcaillau** [control-toolbox/control-toolbox.github.io#37] [Polls]
@ocots @PierreMartinon @joseph-gergaud dear ct owners, please vote!

**Comment by ocots** [comment]
En fait, j'ai voté pour `.org` mais j'aime pareil `.science`. Je pensais qu'on pouvait faire plusieurs choix.

**Comment by ocots** [comment]
Check this: https://docs.github.com/fr/pages/configuring-a-custom-domain-for-your-github-pages-site

**Comment by jbcaillau** [comment]
@ocots @PierreMartinon @joseph-gergaud the previous poll sucks (only one possible vote). sorry about that. here is a new one (majority judgement, of course 🙂):
[control-toolbox domain name](https://app.mieuxvoter.fr/en/votes/wqa-icg-etyn)

**Reply by ocots** [reply]
Bien le jugement majoritaire !
